docs/get-started/cli.md

```diff
@@ -4,4 +4,219 @@ title: On your local machine
 
 # Get started on your local machine
 
-TODO
+Although it takes a little longer to get started, building Packs on your local machine has some advantages over the the web-based editor:
+
+- You can use your own code editing tools, such as [Visual Studio Code](https://code.visualstudio.com/).
+- You can use your own version control system, such as [GitHub](https://github.com).
+- You can use popular JavaScript libraries[^1], such as those in [NPM](https://www.npmjs.com/).
+
+[^1]: Not all libraries available on NPM are compatible with the Packs SDK. See the [Using libraries](../guides/advanced/libraries.md) guide for more information.
+
+Local development is enabled through the `coda` command line tool (CLI). Keep reading to learn how to install the CLI and use it to build a Pack.
+
+## Before you get started
+
+To create a Pack you will need a Coda account, with [Doc Maker access](https://help.coda.io/en/articles/3388781-members-and-roles) in your workspace. If you're new to Coda, [sign up](https://{{coda.domain}}/signup) for a free account and you'll automatically be made a Doc Maker in your personal workspace.
+
+Make sure you have `node` and `npm` available on your machine. These two tools often come bundled together, and more information on how to install them is available in the [npm docs](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+The instructions below assume some familiarity with the terminal / command prompt. If you aren't used to using this interface consult the help material for your operating system.
+
+## Set up the Pack structure
+
+1. Create a directory for your new Pack.
+
+    ```sh
+    mkdir my-pack
+    cd my-pack
+    ```
+
+1. Install the Packs SDK.
```

Comment:
Weird that these are all numbered `1.` but they seem to render as sequentially in the rendered output. Is that Markdown fanciness so you don't have renumber lists as you iterate?